ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/ros2/ROS2BehaviorTreeMessageTools.java

```diff
@@ -33,6 +26,7 @@ public static void clearLists(BehaviorTreeStateMessage treeStateMessage)
    {
       treeStateMessage.getBehaviorTreeTypes().resetQuick();
       treeStateMessage.getBehaviorTreeIndices().clear();
+      treeStateMessage.getPartialDataNodes().clear();
```

Comment:
Partial data is now stored separately in the state messages, no longer exploiting the basic nodes field which was confusing.